lib/cartopy/examples/barbs.py

```diff
@@ -6,6 +6,7 @@
 
 
 def main():
+    fig = plt.figure(figsize=(10, 5))
```

Comment:
The examples I changed used projections that are non-square, so changing the `figsize` helps reduce the extra whitespace.